tests/integ/modin/index/test_reindex.py

```diff
@@ -128,9 +128,7 @@ def test_ordered_index_unordered_new_index():
 @pytest.mark.parametrize("limit", [None, 1, 2, 100])
 @pytest.mark.parametrize("method", ["bfill", "backfill", "pad", "ffill"])
 def test_datetime_with_fill(limit, method):
-    query_count = 2
-    join_count = 2
-    with SqlCounter(query_count=query_count, join_count=join_count):
+    with SqlCounter(query_count=2 if limit is None else 3, join_count=2):
```

Comment:
add a todo for fixing limit once @sfc-gh-rdurrani 's is monotonic PR is done

Reply:
query count is still the same, so no changes required.